source/components/organisms/FormAgent/FormAgent.js

```diff
@@ -91,7 +91,7 @@ class FormAgent extends Component {
             }, true);
         }
 
-        return coniditionsIsValid && answers[question.key] === undefined;
+        return coniditionsIsValid && answers[question.id] === undefined;
```

Comment:
This breaks dependency questions for some reasons, answering "Nej" on partnerSameAddress should output partnerAddress.

Reply:
Alright, checking what causes the break

Reply:
Found the issue, the key property key have been replaced by id on all dependencys :P

Must certain a search and replace for "key" that went to "good".